python-avd/pyavd/_cv/workflows/finalize_workspace_on_cv.py

```diff
@@ -87,3 +101,157 @@ async def finalize_workspace_on_cv(workspace: CVWorkspace, cv_client: CVClient)
         return
 
     return
+
+
+async def process_workspace_build_details(
```

Comment:
Please try to split this out in smaller nicer functions that take care of each part, and then call them from this main function.

Reply:
Fixes. Split logic across 2 additional functions with smaller scope